modules/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/PropagatedSpan.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+/**
+ * Propagated span through context propagation
+ */
+public class PropagatedSpan extends OTelSpan {
```

Comment:
Suggestion, to preserve the  convention with all other classes in this package: `OTelPropagatedSpan`